### ISSUE TITLE
Entity_.__ALL_RELATIONS like __ALL_PROPERTIES

### ISSUE THREAD

**Issue by TKolbusz** [objectbox/objectbox-java#565]
Hi,

it would be great if generated entity class had access to array of RelationInfo like it has access to all properties.
Right now eager loading relations requires
1) explicit list of all relations and QueryBuilder.eager
or 
2) hard reflection loop calling getTarget()

Explicit list __ALL_RELATIONS would ease up creating algorithms like eager loading.

What do you think?

**Comment by greenrobot-team** [comment]
I agree, this would have come in handy when I wrote a SQLite import library as well. Should be an easy generator change.

cc @greenrobot 

-ut
